expertise/service/expertise.py

```diff
@@ -639,7 +628,6 @@ def start_expertise(self, request, client_v1, client):
         self.logger.info('just before submitting')
 
         self.logger.info(f"\nconf: {config.to_json()}\n")
-        self.redis.save_job(config)
```

Comment:
you removed the save here but you are changing the status to QUEUED? and then it is set later in another call, when is the job really queued?